java/ql/test/library-tests/structure/structure/A.java

```diff
@@ -7,7 +7,7 @@ public class A {
 	int y;
 	class MemberClass { }
 	void m() {
-		class LocalClass { int n; }
+		class LocalClass { int n; void localClassMethod() { class DoublyLocalClass { void doublyLocalClassMethod() { } } } }
```

Comment:
No, this is relating to the commit https://github.com/github/codeql/pull/9876/commits/b763c406b68d83dbc9f7400be854b544b3a741c8 which was necessary to avoid a spurious test failure